tests/test_plugin.py

```diff
@@ -333,18 +341,14 @@ def test_menu_cache_invalidate_after_menucontent_publish(self):
             placeholder, menu_content.menu, "menu/menu.html"
         )
 
-        # Now publish the page content containing the plugin,
-        # so the page can be viewed
-        version = page_content.versions.get()
-        version.publish(self.get_superuser())
-
         # Making sure there is no cachekey existed before rendering page
         cache_key = CacheKey.objects.all().count()
         self.assertEqual(cache_key, 0)
 
-        # And view the page
```

Comment:
I'm sure that this test should work because the version was published and the endpoint was a published endpoint!